lambdas/es/indexer/document_queue.py

```diff
@@ -43,6 +43,12 @@
 RETRY_429 = 3
 
 
+def get_id(key, version_id):
+    """guarantee primary key uniqueness for package delete (marker) documents"""
```

Comment:
The comment seems a bit misleading since it guarantees uniqueness for all S3 objects.